filters/include/pcl/filters/passthrough.h

```diff
@@ -76,7 +77,7 @@ namespace pcl
     * \author Radu Bogdan Rusu
     * \ingroup filters
     */
-  template <typename PointT>
+  template <typename PointT, typename Executor = inline_executor<oneway_t, single_t, blocking_t::always_t>>
```

Comment:
Am I missing something, or is there no CRTP here, except the old design?

This design is bad because it goes against delaying execution till the `filter()` is called

Reply:
Fixed.